plugins/cpu/fu-cpu-helper-cet-common.c

```diff
@@ -9,20 +9,29 @@
 
 #include "fu-cpu-helper-cet-common.h"
 
+#if defined(__has_attribute)
+#if !__has_attribute(__noclone__)
+#define ATTRIBUTE_NOCLONE
+#endif
+#endif
+#ifndef ATTRIBUTE_NOCLONE
+#define ATTRIBUTE_NOCLONE __attribute__((__noclone__))
+#endif
+
 static void
 fu_cpu_helper_cet_testfn_fptr (void)
 {
 }
 
 static void
-__attribute__ ((noinline, noclone))
+__attribute__((__noinline__)) ATTRIBUTE_NOCLONE
```

Comment:
?? Got any background?

Reply:
I think it specifically needs noinline, noclone for CET to be detected... Maybe drop this bit and we can check it in another PR?

Reply:
yeah I'll move it over to another PR.  If noclone is needed, then we probably need to turn off CET checker when using clang.